cocoa/BugsnagReactNative.m

```diff
@@ -230,7 +232,9 @@ + (void)startWithConfiguration:(BugsnagConfiguration *)config {
                             withValue:codeBundleId
                         toTabWithName:@"app"];
     }
-    if (![Bugsnag bugsnagStarted]) {
+    if ([Bugsnag bugsnagStarted] && !config.autoNotify) {
+        bsg_kscrash_setHandlingCrashTypes(BSG_KSCrashTypeUserReported);
+    } else if (![Bugsnag bugsnagStarted]) {
         [Bugsnag startBugsnagWithConfiguration:config];
```

Comment:
It appears that for this clause, fatal exceptions will still be reported even if `config.autoNotify` is false